tubular/tests/test_retire_one_learner.py

```diff
@@ -2,22 +2,19 @@
 Test the retire_one_learner.py script
 """
 
-from mock import patch, DEFAULT
-
 from click.testing import CliRunner
-from slumber.exceptions import HttpNotFoundError
-
-from tubular.scripts.retire_one_learner import (
-    END_STATES,
-    ERR_BAD_CONFIG,
-    ERR_BAD_LEARNER,
-    ERR_SETUP_FAILED,
-    ERR_UNKNOWN_STATE,
-    ERR_USER_AT_END_STATE,
-    ERR_USER_IN_WORKING_STATE,
-    retire_learner
-)
-from tubular.tests.retirement_helpers import fake_config_file, get_fake_user_retirement
+from mock import DEFAULT, patch
+
+from tubular.exception import HttpDoesNotExistException
+from tubular.scripts.retire_one_learner import (END_STATES, ERR_BAD_CONFIG,
```

Comment:
long line splitting is recommended to be done differently please take a look at a python style guide https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#breaking-long-lines